.envs/.production.example/.api

```diff
@@ -1,6 +1,5 @@
 # API (Django)
 # ------------------------------------------------------------------------------
-DATABASE_HOST=db
```

Comment:
Sure this is not needed anymore?

Reply:
Ah because it's now set in the compose file. Got it

Reply:
Yeah, before for dev it was unset, which is no problem, because we override the `CMD`. But I ran into it, when locally testing Gunicorn. So it made sense to fix it for good :)